setup.py

```diff
@@ -3,68 +3,74 @@
 from sys import version_info
 
 
-install_requires = [
-    'six',
-    'content-io >= 1.2.5',
-    'simplejson >= 3.2.0'
-]
+install_requires = ["six", "content-io >= 1.2.5", "simplejson >= 3.2.0"]
 
 tests_require = [
-    'coverage',
-    'Markdown <= 2.4.1',
-    'Pillow <= 3.4.2',
+    "coverage",
+    "Markdown <= 3.3.0",
+    "Pillow <= 3.4.2",
 ]
 
 if version_info < (3,):
-    tests_require += ['unittest2']
+    tests_require += ["unittest2"]
 
-version = __import__('djedi').__version__
+version = __import__("djedi").__version__
 
 setup(
-    name='djedi-cms',
+    name="djedi-cms",
     version=version,
-    description='Django content management as it should be',
+    description="Django content management as it should be",
     long_description=(
-        '.. image:: https://djedi-cms.org/_static/djedi-portrait.svg\n\n'
-        '- Read the documentation_\n'
-        '- Browse the source_\n\n'
-        '.. image:: https://travis-ci.org/5monkeys/djedi-cms.svg?branch=master\n'
-        '    :target: https://travis-ci.org/5monkeys/djedi-cms\n'
-        '.. image:: https://coveralls.io/repos/5monkeys/djedi-cms/badge.svg?branch=master\n'
-        '    :target: https://coveralls.io/r/5monkeys/djedi-cms?branch=master\n\n'
-        '.. _documentation: https://djedi-cms.org/\n'
-        '.. _source: https://github.com/5monkeys/djedi-cms\n\n'
+        ".. image:: https://djedi-cms.org/_static/djedi-portrait.svg\n\n"
+        "- Read the documentation_\n"
+        "- Browse the source_\n\n"
+        ".. image:: https://travis-ci.org/5monkeys/djedi-cms.svg?branch=master\n"
+        "    :target: https://travis-ci.org/5monkeys/djedi-cms\n"
```

Comment:
This link wont be relevant so let's drop it and add a proper github actions link later